torch/distributed/_sharded_tensor/api.py

```diff
@@ -245,3 +346,18 @@ def size(self) -> torch.Size:
         Returns the size of the self tensor. The returned value is a subclass of tuple.
         """
         return torch.Size(self._dims)
+
+    def _register_remote_shards(self, remote_shards: List[rpc.RRef]):
```

Comment:
Ditto

Reply:
Another question is why is this a list instead of a dict keyed by name/rank? How shall we use it?

Reply:
Good point, I'll change this to dict for easy of use.